code/_onclick/hud/human.dm

```diff
@@ -62,6 +62,27 @@
 /obj/screen/devil/soul_counter/proc/clear()
 	invisibility = INVISIBILITY_ABSTRACT
 
+obj/screen/dwarf
+	invisibility = INVISIBILITY_ABSTRACT
+
+/obj/screen/dwarf/alcohol_display
+	icon = 'icons/mob/screen_gen.dmi'
+	icon_state = "power_display2"
+	name = "alcohol stored"
+	screen_loc = ui_alcohol_amount
+
+/obj/screen/dwarf/alcohol_display/proc/update(mob/living/carbon/human/usr)
+	var/mob/living/carbon/human/H = usr
+	var/obj/item/organ/alcoholvessel/dwarf
+	dwarf = H.getorganslot("dwarf_organ")

```

Comment:
Ideally this HUD element would only be created if the person was a dwarf (in the organ's ```Insert()``` proc)